app/assets/javascripts/active_admin.js

```diff
@@ -0,0 +1,60 @@
+//= require active_admin/base
+
+//= require froala_editor.pkgd.min.js
+//= require plugins/image.min.js
+
+// $(function() {
```

Comment:
In the future when editing .js files, I don't believe for "regular" javascript you need to prefix lines this way: the //= is merely telling Rails to, before it renders the file, process any rails-specific keywords (e.g. "require") and then continues on including the javascript. 

If this was actually supposed to be commented out / deleted though, let me know and I'll do that instead of unprexing each line.

Reply:
Got it, those weren't meant to be commented out so thank you!